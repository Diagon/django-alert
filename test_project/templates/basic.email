{% load alert_email_tags %}

{% p %}Hello {{ username }},{% endp %}

{% p %}I'm writing you to say hello.{% endp %}

{% h1 %}Main Points{% endh1 %}

{% p %}There is only {% a href='http://google.com' %}one{% enda %} point, actually.{% endp %}

{% p %}Best,
James{% endp %}