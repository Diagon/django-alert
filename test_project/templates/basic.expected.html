<p>Hello Alex,</p>

<p>I'm writing you to say hello.</p>

<h1>Main Points</h1>

<p>There is only <a href="http://google.com">one</a> point, actually.</p>

<p>Best,<br>James</p>